pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion = "2.0.1-dev"
+	bundleVersion = "2.0.1-dev-rossf7"
```

Comment:
Probably different subfix would be better, (e.g. `-patch1`)

Reply:
This is me testing. I will remove before merging.